### ISSUE TITLE
BE | Profile | Auth Exp | Datadog Access Discovery

### ISSUE THREAD

**Issue by tpharrison** [department-of-veterans-affairs/va.gov-team#54053]
## Background
The Authenticated Experience team would like access to Datadog in order to aid our troubleshooting efforts. Our goal is to improve monitoring and alerting of high-severity errors and latency issues, so that we can resolve them more quickly.

As of February 21, 2023, Read-only (RO) access to Datadog is available to VFS Teams beginning. VFS Team members will be able to access Datadog and view dashboards and monitors.

Documentation will be made available on the Platform Website covering the following topics:
- Datadog overview - Welcome, here’s what to expect
- Dashboards and monitors - a list of pre-configured dashboards with health and performance data for major backend applications (ie Vets-API and related services) with accompanying explainer info!
- How to request access to Datadog (for new VFS Team members, post-release)
- How will VFS teams get access?

VFS team members can expect to receive an invite to Okta from DOTS the week of 2/21. DOTS Okta is the single-sign-on solution used to access tools provided by DOTS

After this release, new VFS team members will request access to Datadog through the Platform Website or through #vfs-platform-support

## Tasks
- [x] Verify access through OKTA DOTS. 
- [x] Review Dashboards and Monitors.
- [x] Request access for Derrick Ellerbie.
- [x] Document list of dashboards/monitors that are needed (if any) beyond what is provided.
- [ ] Determine if our team will be granted write access to be able to create our own dashboards/monitors.

## Acceptance Criteria
- [ ] Review findings with team

**Comment by tpharrison** [comment]
@Samara-Strauss @BerniXiongA6 
FYI - after submitting a request through Max.gov, I have received my invite from DVA Okta and I'm now able to access Datadog.

**Comment by tpharrison** [comment]
@Samara-Strauss @BerniXiongA6 

Datadog currently has pre-configured dashboards and health and performance metrics available, but most of these are at the infrastructure-level and not specific to Profile or My VA endpoints.

So we'll need write access in order to create our own dashboard, monitors, alerts, etc.  I reached out to Jesse House and asked about getting write access.  His response...

> _Hey Tom, thanks for reaching out. We’re putting a plan together for write access. Your name and team are definitely at the top of our list. I should have more info this week. I’ll be sure to follow up soon._

I've opened [this epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/55009) to help keep track of things once we get write access.  I'll create the associated tickets once we get write access.

Let me know if you have any questions.
